test/jdk/javax/swing/JComboBox/EditableComboBoxPopupPos.java

```diff
@@ -0,0 +1,139 @@
+                    String[] comboStrings = {"One", "Two", "Three"};
+                    cb1 = new JComboBox(comboStrings);
+                    cb1.setEditable(true);
+                    cb1.setBorder(BorderFactory.createTitledBorder(
+                            "Editable JComboBox"));
+
+                    cb2 = new JComboBox(comboStrings);
+                    cb2.setEditable(true);
+
+                    panel.add(cb1);
+                    panel.add(cb2);
+
+                    frame = new JFrame();
+                    frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+                    frame.add(panel);
+                    frame.pack();
+                    frame.setLocationRelativeTo(null);
+                    frame.setVisible(true);
+                });
+
+                // Change starting selection to check if the position of the
+                // first selection item is in the correct position on screen.
+                cb1.setSelectedIndex(1);
+                cb2.setSelectedIndex(1);
+
+                runTestOnComboBox(cb1);
+                runTestOnComboBox(cb2);
+
+                checkSelection(cb1, cb2);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            SwingUtilities.invokeAndWait(() -> frame.dispose());
+        }
+    }
+
+    private static void setLookAndFeel(UIManager.LookAndFeelInfo laf) {
+        try {
+            UIManager.setLookAndFeel(laf.getClassName());
+        } catch (UnsupportedLookAndFeelException ignored){
+            System.out.println("Unsupported LookAndFeel: " + laf.getClassName());
+        } catch (ClassNotFoundException | InstantiationException |
+                IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static void runTestOnComboBox(JComboBox cb) {
+        robot.mouseMove(cb.getLocationOnScreen().x + cb.getWidth()
+                - BUTTON_OFFSET, cb.getLocationOnScreen().y
+                + POPUP_OFFSET);
+        robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseMove(cb.getLocationOnScreen().x
+                        + (cb.getWidth() / 2) - BUTTON_OFFSET,
+                cb.getLocationOnScreen().y + cb.getHeight()
+                        + POPUP_OFFSET);
+        robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+    }
+
+    private static void checkSelection(JComboBox c1, JComboBox c2)
+            throws InterruptedException, InvocationTargetException {
+        if (c1.getSelectedItem().toString().equals("One")
+                && c2.getSelectedItem().toString().equals("One")) {
+            System.out.println(lafName + " Passed");
+            SwingUtilities.invokeAndWait(() -> frame.dispose());
+        } else {
+            SwingUtilities.invokeAndWait(() -> frame.dispose());
```

Comment:
Your `main` method disposes of the frame. You should just move try-finally block into the for-loop over available Look-and-Feels.

Reply:
Added to the bottom of the loop